src/UnityExtension/Assets/Editor/GitHub.Unity/GitHub.Unity.asmdef

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "GitHub.Unity",
+    "references": [],
+    "includePlatforms": [
+        "Editor"
+    ],
+    "excludePlatforms": []
```

Comment:
We talked about this quickly in person. Can you remind my brain what this is for?

Reply:
Unity will compile all files in a directory with a asmdef in their own DLL (UnityTests are then compiled in a separate DLL from this one). It's not really very relevant, we have our own csproj files for building the DLLs, but it's nice to test.